README.md

```diff
@@ -53,9 +49,9 @@ ssl_crt: |
   -----END CERTIFICATE-----
 ```
 
-* Install or update ansible playbooks using [librarian-ansible](https://github.com/bcoe/librarian-ansible):
+* Install or update ansible playbooks using [ansible-galaxy](http://docs.ansible.com/galaxy.html):
 ```
-bundle exec librarian-ansible install
+ansible-galaxy install -r requirements.yml
```

Comment:
I agree that we should fix the Makefile so that everyone can use it. For now, refer to the contents, and I'll talk to Brett about raising a story.